db/migrate/20180523162842_add_is_counselor_to_educator.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsCounselorToEducator < ActiveRecord::Migration[5.1]
+  def change
+    add_column :educators, :is_counselor, :boolean, default: false
```

Comment:
There's a `staff_type` field that has some "Counselor" values if I remember right, that might be worth checking out.

If not, this [EducatorLabel](https://github.com/studentinsights/studentinsights/blob/master/app/models/educator_label.rb#L1.) class might be useful for this purpose, it's what I used to label whether folks are on the NGE/10GE teams.